pom.xml

```diff
@@ -325,6 +326,20 @@
                     </configurator>
                   </action>
                 </pluginExecution>
+                <!-- m2eclipe does not support fmt-maven-plugin -->
+                <pluginExecution>
+                  <pluginExecutionFilter>
+                    <groupId>com.coveo</groupId>
+                    <artifactId>fmt-maven-plugin</artifactId>
+                    <versionRange>[0.0,)</versionRange>
+                    <goals>
+                      <goal>check</goal>
+                    </goals>
+                  </pluginExecutionFilter>
+                  <action>
+                    <ignore />
```

Comment:
Would it make sense actually execute the `check` goal?  Could replace this `<action>` clause with:
```
             <action>
               <execute>
                 <runOnIncremental>false</runOnIncremental>
               </execute >
             </action>
```